### ISSUE TITLE
Build failure: `linuxKernel.packages.linux_6_8.vhba`

### ISSUE THREAD

**Issue by Jayman2000** [NixOS/nixpkgs#295717]
### Steps To Reproduce

1. Run `nix-build -E '(import <nixpkgs> { }).linuxKernel.packages.linux_6_8.vhba'`

### Build log

<https://gist.github.com/Jayman2000/5c2cf9b79db33b105fd42ece5738e0e5>

### Additional context

This bug is making my NixOS config fail to build. Linux 6.8 is the latest version of Linux at the moment, and my NixOS config does this:

```nix
{ pkgs, ... }:

{
  boot.kernelPackages = pkgs.linuxPackages_latest;
}
```

I was able to work around this bug by doing this:

```nix
{ pkgs, ... }:

{
  boot.kernelPackages = pkgs.linuxKernel.packages.linux_6_7;
}
```

### Notify maintainers

@bendlas

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.7.9, NixOS, 23.11 (Tapir), 23.11.5286.ddcd7598b218`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(root): `"nixos-23.11"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

$ 
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by bendlas** [comment]
Module version can be updated. That will probably resolve it.
https://cdemu.sourceforge.io/project/#binary

**Comment by romatthe** [comment]
Having this issue as well. I'm on the latest kernel on `23.11` (6.8.1). The VHBA module fails to build. The module update has landed in unstable but hasn't been backported to `23.11` yet. Since I need CDEmu for now, I'm forced to stick to 6.7 at the moment.

Not the end of the world, but slightly annoying that some kernel modules on stable are broken right now.

**Comment by Jayman2000** [comment]
This was fixed by #285972 and #305046.
